src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -1613,7 +1613,7 @@ TEST_P(conv_scale_activation_eltwise_fp32_quantize_i8, basic) {
         convolution("conv", input_info("input"), { "weights" }, { "bias" }, p.groups, p.stride, p.pad, p.dilation),
         data("scale_data", get_mem(get_per_channel_layout(p))),
         eltwise("scale", { input_info("conv"), input_info("scale_data") }, eltwise_mode::prod),
-        activation("activation", input_info("scale"), activation_func::hyperbolic_tan),
+        activation("activation", input_info("scale"), activation_func::relu),
```

Comment:
Sorry, but we added separate number for onednn fused case. Can we use it to keep hyperbolic_tan?

Reply:
Done